x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/loadingservice/LocalModel.java

```diff
@@ -135,4 +134,39 @@ public void infer(Map<String, Object> fields, InferenceConfigUpdate update, Acti
         }
     }
 
+    public long acquire() {
```

Comment:
package private? The only way to "get" a trained model object should be through a known code path. so, we should restrict their construction and restrict this method as well.